h/renderers.py

```diff
@@ -2,11 +2,31 @@
 
 from __future__ import unicode_literals
 
+try:
+    from StringIO import StringIO # python 2
+except ImportError:
+    from io import StringIO # python 3
+
 import pyramid.renderers
 
 
 json_sorted_factory = pyramid.renderers.JSON(sort_keys=True)
 
 
+class SVGRenderer(object):
+    def __init__(self, info):
+        pass
+
+    def __call__(self, value, system):
+        request = system.get('request')
+        if request is not None:
+            response = request.response
+            ct = response.content_type
+            if ct == response.default_content_type:
+                response.content_type = 'image/svg+xml'
+
+        return value
```

Comment:
This is based on the `CSVRenderer` here: <https://docs.pylonsproject.org/projects/pyramid-cookbook/en/latest/templates/customrenderers.html>